src/Menu/Register.java

```diff
@@ -0,0 +1,71 @@
+package Menu;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+public class Register extends Client{
+    Scanner sc = new Scanner(System.in);
+
+    public Register() {
+        super();
+    }
+
+    public Register(String name, Date birthDate, String userCPF, String email, Date registerDate, String userAddres) {
+        super(name, birthDate, userCPF, email, registerDate, userAddres);
+    }
+
+    public boolean validatorCPF(String cpf) {
```

Comment:
```suggestion
    public boolean isCPFValid(String cpf) {
```

Normalmente quando vc quer fazer uma função de validação, se coloca um `is` no começo e devolve um boolean. Semanticamente, fica algo como "o CPF é valido? sim/não"